src/main/java/uk/gov/hmcts/reform/preapi/controllers/MediaServiceController.java

```diff
@@ -199,4 +214,25 @@ public ResponseEntity<CaptureSessionDTO> startLiveEvent(@PathVariable UUID captu
 
         return ResponseEntity.ok(captureSessionService.startCaptureSession(captureSessionId, ingestAddress));
     }
+
+    @PostMapping("/generate-asset")
+    @Operation(
+        operationId = "generateAsset",
+        summary = "LEGACY - Given a source & destination, this endpoint will generate a streaming asset for a given mp4"
+    )
+    @Parameter(
+        name = "code",
+        description = "Rudimentary security code to prevent unauthorised access to this endpoint",
+        schema = @Schema(implementation = String.class)
+    )
```

Comment:
Does this mean it needs to be ignored by the X-User-Id auth checker?

Reply:
yes it does as it'll get called as part of the manual editing process which won't have an X-User-Id
Is there something I need to change in addition to get that to work

Reply:
Yeah- you just need to add the endpoint to this list: https://github.com/hmcts/pre-api/blob/master/src/main/java/uk/gov/hmcts/reform/preapi/config/SecurityConfig.java#L24-L38